### ISSUE TITLE
Server-side rendering throws: React DOM tree root should always have a node reference

### ISSUE THREAD

**Issue by jagged3dge** [codecks-io/react-reform#17]
Hello!

I really appreciate the work put into this library. I believe everything is working fine when working client-side only. I just copy-pasted example code from [Custom button text](http://react-reform.codecks.io/recipes/#custom-button-text) documentation page into my SSR-enabled project. Rendering the page on the server makes React throw: ```React DOM tree root should always have a node reference.```

Full stack-trace:
```
Invariant Violation: React DOM tree root should always have a node reference.
    at invariant (D:\Work\SSRproj\ui\node_modules\fbjs\lib\invariant.js:38:15)
    at Object.getNodeFromInstance (D:\Work\SSRproj\ui\node_modules\react\lib\ReactDOMComponentTree.js:164:67)
    at Object.ReactDOMInput.updateWrapper (D:\Work\SSRproj\ui\node_modules\react\lib\ReactDOMInput.js:153:71)
    at ReactDOMComponent.Mixin.updateComponent (D:\Work\SSRproj\ui\node_modules\react\lib\ReactDOMComponent.js:691:23)
    at ReactDOMComponent.Mixin.receiveComponent (D:\Work\SSRproj\ui\node_modules\react\lib\ReactDOMComponent.js:668:10)
    at Object.ReactReconciler.receiveComponent (D:\Work\SSRproj\ui\node_modules\react\lib\ReactReconciler.js:127:22)
    at [object Object].ReactCompositeComponentMixin._updateRenderedComponent (D:\Work\SSRproj\ui\node_modules\react\lib\ReactCompositeComponent.js:775:23)
    at [object Object].ReactCompositeComponentMixin._performComponentUpdate (D:\Work\SSRproj\ui\node_modules\react\lib\ReactCompositeComponent.js:753:10)
    at [object Object].ReactCompositeComponentMixin.updateComponent (D:\Work\SSRproj\ui\node_modules\react\lib\ReactCompositeComponent.js:672:12)
    at [object Object].ReactCompositeComponentMixin.receiveComponent (D:\Work\SSRproj\ui\node_modules\react\lib\ReactCompositeComponent.js:571:10)
    at Object.ReactReconciler.receiveComponent (D:\Work\SSRproj\ui\node_modules\react\lib\ReactReconciler.js:127:22)
    at Object.ReactChildReconciler.updateChildren (D:\Work\SSRproj\ui\node_modules\react\lib\ReactChildReconciler.js:85:25)
    at ReactDOMComponent.ReactMultiChild.Mixin._reconcilerUpdateChildren (D:\Work\SSRproj\ui\node_modules\react\lib\ReactMultiChild.js:189:32)
    at ReactDOMComponent.ReactMultiChild.Mixin._updateChildren (D:\Work\SSRproj\ui\node_modules\react\lib\ReactMultiChild.js:288:31)
    at ReactDOMComponent.ReactMultiChild.Mixin.updateChildren (D:\Work\SSRproj\ui\node_modules\react\lib\ReactMultiChild.js:276:12)
    at ReactDOMComponent.Mixin._updateDOMChildren (D:\Work\SSRproj\ui\node_modules\react\lib\ReactDOMComponent.js:883:12)
    at ReactDOMComponent.Mixin.updateComponent (D:\Work\SSRproj\ui\node_modules\react\lib\ReactDOMComponent.js:712:10)
    at ReactDOMComponent.Mixin.receiveComponent (D:\Work\SSRproj\ui\node_modules\react\lib\ReactDOMComponent.js:668:10)
    at Object.ReactReconciler.receiveComponent (D:\Work\SSRproj\ui\node_modules\react\lib\ReactReconciler.js:127:22)
    at [object Object].ReactCompositeComponentMixin._updateRenderedComponent (D:\Work\SSRproj\ui\node_modules\react\lib\ReactCompositeComponent.js:775:23)
    at [object Object].ReactCompositeComponentMixin._performComponentUpdate (D:\Work\SSRproj\ui\node_modules\react\lib\ReactCompositeComponent.js:753:10)
    at [object Object].ReactCompositeComponentMixin.updateComponent (D:\Work\SSRproj\ui\node_modules\react\lib\ReactCompositeComponent.js:672:12)
```

**Comment by danielberndt** [comment]
Thanks for letting me know. I'll have a look. Probably related issue/solution: https://github.com/nkbt/react-interval/pull/9

**Comment by danielberndt** [comment]
should be fixed with v1.1.1.
Let me know if it works for you!
